pkg/plugin/plugin.go

```diff
@@ -43,3 +44,19 @@ func (l *Loader) Get(name string) Plugin {
 
 	return l.plugins[name]
 }
+
+// for some reasons mapstructure.Decode() gives empty arrays for all resulting config fields
+// this is quick workaround hack t make it work
+// TODO: investigate and fix mapstructure.Decode() behaviour and remove this dirty hack
```

Comment:
Just as a info, `FIXME: ` label is commonly used as well instead of `TODO:` for some buggy cases